stratum/hal/lib/tdi/es2k/es2k_hal.h

```diff
@@ -81,7 +82,8 @@ class Es2kHal final {
                                   SwitchInterface* switch_interface,
                                   AuthPolicyChecker* auth_policy_checker,
                                   absl::Notification* ready_sync = nullptr,
-                                  absl::Notification* done_sync = nullptr)
+                                  absl::Notification* done_sync = nullptr,
+                                  const TargetOptions* target_options = nullptr)
```

Comment:
const TargetOptions* target_options =TargetOptions::default_target_options?
Since TargetOptions::default_target_options is created to be the default?